testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -456,8 +456,11 @@ class MiniAppDisplayActivity : BaseActivity(), PreloadMiniAppWindow.PreloadMiniA
                     NativeEventType.EXTERNAL_WEBVIEW_CLOSE,
                     "External webview closed"
                 )
-                if (!isClosedByBackPressed)
+                if (!isClosedByBackPressed) {
                     sampleWebViewExternalResultHandler.emitResult(intent)
+                } else {
+                    finish()
```

Comment:
is this the correct behaviour? we are opening an external url from miniapp to a external webview, closing the external webview should not close the miniapp it self. it should just close the external webview and come back to the miniapp screen. that's why there was not any `else` statement. please revert this to the original one.

Reply:
from this screen: https://rak.box.com/s/8su7ea88scov85798ztduv0moge800me
assuming you have checked the [issue video,](https://rak.app.box.com/s/n98c8n6baqpiucotfn9bpzgloydfa33w), if the issue is still a bug, then instead I made some arguments to make sure that it's from MiniAppByUrlActivity. Please check.

Reply:
@shohiebsense i have checked the video, looks like it behaved correctly, the url qa is opening is somehow not correct or a `Link` deeplink, that's why it opened the miniapp first(which failed to load)then miniapp opens the google play store in a external webview. So closing the external webview should come back to the miniapp which is failed to load so showing a empty screen, then back should close the miniapp screen.

Reply:
@munir-rakuten the url is provided by QA should be correct. I verify myself, this is it https://rak.box.com/s/0ai07421oibuykjl6ca905uiopo5tsq8 if the activity is not finished.

Reply:
@shohiebsense the url must be a miniapp url to load, not a `google.com`, if the url is not a miniapp url it will open the external webview, which is happening in your video, if external webview back close the miniapp also then when we will open a link from miniapp it will be affected as well.

Reply:
This url https://one.rakuten.co.jp/miniapp/preview/404e46b4-263d-4768-b2ec-8a423224bead is not correct url for mini app? @munir-rakuten

Reply:
`one.rakuten.co.jp` is not a valid url for demo app. it's a deeplink is for link app and it's a deeplink, `mini app by url` should not open deeplinks.